src/components/SideMenu/Tab.tsx

```diff
@@ -0,0 +1,19 @@
+import './Tab.styles.scss';
```

Comment:
Also tab could be called `SideMenuTab` or `MenuTab`

Reply:
done